client/src/constants.js

```diff
@@ -2,10 +2,12 @@
 import districtJson from './slovakiaData/district-centers.json'
 import regionJson from './slovakiaData/region-centers.json'
 import districtCityJson from './slovakiaData/district_cities.json'
+import allCityJson from './slovakiaData/all_cities.json'
 
 export const SLOVAKIA_DISTRICT = districtJson
```

Comment:
So, this isn't used anymore? Or it never was? If so, please remove, I think it's related to this.

Reply:
I mean remove both this const (and import) and related file.